client/src/utils/ajax.ts

```diff
@@ -26,6 +26,8 @@ const defaultOptions: RequestInit = {
 function getCSRFToken() {
   const token =
     typeof window !== 'undefined' ? cookies.get('csrf_token') : null;
+
+  console.log(token);
```

Comment:
```suggestion
```

Seeing a variable called `token` get logged is scary no matter how harmless it really is 😅

Reply:
Yep, there should be an eslint rule for this!